diff --git a/redux/api/user.ts b/redux/api/user.ts
index 6dbe601..dd58905 100644
--- a/redux/api/user.ts
+++ b/redux/api/user.ts
@@ -55,17 +55,23 @@ export const userApi = createApi({
       }),
     }),
     markAllRead: builder.mutation({
-      query: (id) => ({
+      query: () => ({
         url: `customer/notifications/mark-all-read`,
         method: "PUT",
       }),
     }),
     deleteAllNotification: builder.mutation({
-      query: (id) => ({
+      query: () => ({
         url: `customer/notifications/delete-all`,
         method: "DELETE",
       }),
     }),
+    deleteSingleNotification: builder.mutation({
+      query: (id) => ({
+        url: `customer/notifications/${id}`,
+        method: "DELETE",
+      }),
+    }),
   }),
 });
 
@@ -77,6 +83,7 @@ export const {
  useLazyGetNotificationsQuery,
  useMarkReadMutation,
  useMarkAllReadMutation,
- useDeleteAllNotificationMutation
+ useDeleteAllNotificationMutation,
+ useDeleteSingleNotificationMutation
 
 } = userApi;
